helm/loginapp/templates/configmap.yaml

```diff
@@ -7,4 +7,16 @@ metadata:
     {{- include "loginapp.labels" . | nindent 4 }}
 data:
   config.yaml: |
-{{ toYaml (mergeOverwrite $config .Values.configOverwrites) | nindent 4 }}
\ No newline at end of file
+{{ toYaml (mergeOverwrite $config .Values.configOverwrites) | nindent 4 }}
+---
+{{- if and .Values.templates.error .Values.templates.token}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "loginapp.fullname" . }}-templates
+  labels:
+    {{- include "loginapp.labels" . | nindent 4 }}
+data:
```

Comment:
* conditional include, or a loop over all `templates` keys
* should support multi-line (`|-` & `| nindent 4`)